crypto/armor_test.go

```diff
@@ -80,7 +83,11 @@ func TestArmorUnarmorPubKey(t *testing.T) {
 
 	err := depinject.Inject(depinject.Configs(
 		configurator.NewAppConfig(),
-		depinject.Supply(log.NewNopLogger()),
+		depinject.Supply(log.NewNopLogger(),
```

Comment:
default bindings would be useful here too: https://github.com/cosmos/cosmos-sdk/issues/16727.
I'll have a look into implement that.